src/kubernetes-jsonnet/grafana/configs/dashboard-definitions/kubernetes-kubelet-dashboard.jsonnet

```diff
@@ -0,0 +1,97 @@
+local grafana = import "grafonnet/grafana.libsonnet";
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local row = grafana.row;
+local singlestat = grafana.singlestat;
+local template = grafana.template;
+
+local podsCount = singlestat.new(
```

Comment:
Seems like this could even be abstracted into the `numbersinglestat` with an additional function bound to it, maybe `withSparkline()`, what do you think?